extensions.html

```diff
@@ -209,7 +209,7 @@ <h2><dfn>GamepadHapticActuator</dfn> Interface</h2>
           </p>
           <p>
             If the effect type is not compatible with the actuator,
-            <code>playEffect()</code> must immediately return a Promise resolved
+            the Promise returned by <code>playEffect()</code> must be resolved
```

Comment:
This still somewhat implies that when playEffect() returns the promise has already been resolved. To clarify this I would start above by saying, "It returns a new Promise `promise` and performs the following steps in parallel." Then below list out the available failure cases which all "resolve `promise` with..." and the final successful case which "resolves `promise` with `'complete'`."